recipes/default.rb

```diff
@@ -98,5 +98,5 @@
 
 service 'sssd' do
   supports :status => true, :restart => true, :reload => true
-  action [:enable]
+  action node['sssd']['service_actions']
```

Comment:
No, the `supports` part tells Chef how the service can be used, e.g. if it doesn't support `restart`, then Chef will run `stop` then `start` instead. It uses `update-rc.d` for enabling a service.